indico/web/client/js/custom_elements/ind_bypass_block_links.js

```diff
@@ -0,0 +1,27 @@
+// This file is part of Indico.
+// Copyright (C) 2002 - 2023 CERN
+//
+// Indico is free software; you can redistribute it and/or
+// modify it under the terms of the MIT License; see the
+// LICENSE file for more details.
+
+import './ind_bypass_block_links.scss';
+
+customElements.define(
+  'ind-bypass-block-links',
+  class extends HTMLElement {
+    connectedCallback() {
+      window.addEventListener('DOMContentLoaded', () => {
+        const $$bypassBlockTargets = document.querySelectorAll('[id][data-bypass-target]');
```

Comment:
hmm not sure if I like the `$` and `$$`. in older (jquery-based) code we often used `$foo` for variables holding jquery objects, so maybe this can get a bit confusing.

personally I'd not use any prefix for element / element collection variables

Reply:
In my experience with vanilla code, I find it very useful to distinguish between DOM elements and other values, especially since I may use similar names (e.g., `user` to mean user data, and `$user` to mean a DOM element related to such data). What do you thik?